test-data/unit/check-flags.test

```diff
@@ -515,3 +515,59 @@ x, y = 1, 2  # type: Unchecked, Unchecked
 [out]
 main:4: error: Type of variable becomes "Any" due to an unfollowed import
 main:6: error: A type on this line becomes "Any" due to an unfollowed import
+
+[case testDisallowAnyExprUnannotatedFunction]
+# flags: --disallow-any=expr
+def g(s):
+    return s  # E: Expression has type "Any"
```

Comment:
Sorry for not noticing this earlier, but I don't think we should give this warning in unannotated functions.  Should be easy to fix, though!